### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nitschis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+- From within Kodi, navigate to "Add-on Browser"
+- Select "Install from Repository"
+- Choose "Kodi Jellyfin Add-ons", followed by "Video Add-ons"
+- Select the JellyCon add-on and choose install
+
+#### 3. Login
+
+- Within a few seconds after the installation you should be prompted for your server details.
+- If a Jellyfin server is detected on your local network, it will displayed in a dialog. Otherwise, you will be prompted for a URL
+- Select a user from the list, or Manual Login to type in a username/password
```
[the remark this answers is not in the document]
Comment (nitschis) [reply README.md:26]: I don't have a problem editing the file to your wishes. The difficulty is, that I am not very sufficient in git. So the best path for the edits is not clear to me.

[the remark this answers is not in the document]
Comment (nitschis) [reply README.md:26]: Do I do a full on new pull request?

[the remark this answers is not in the document]
Comment (TrueTechy) [reply README.md:26]: You can make your changes on your existing branch, and push those up. Then they appear on this PR, no need for a new PR

[the remark this answers is not in the document]
Comment (nitschis) [reply README.md]: Did the push work?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+- Select "Install from Repository"
+- Choose "Kodi Jellyfin Add-ons", followed by "Video Add-ons"
+- Select the JellyCon add-on and choose install
+
+#### 3. Login
+
+- Within a few seconds after the installation you should be prompted for your server details.
+- If a Jellyfin server is detected on your local network, it will displayed in a dialog. Otherwise, you will be prompted to enter the URL of your Jellyfin server
+- A "Quick Connect" code will be displayed. Which can be used to Login without the need of credentials, once you enter it on your Jellyfin server under "User Settings" -> "Quick Connect"
+- Alternatively you can select "Manual Login" to select a user from the list, or Manual Login using your username and password
```
Comment (mcarlton00) [inline README.md:26]:
```suggestion
- If Quick Connect is enabled in the server, a code will be displayed that you can use to log in via Quick Connect in the web UI or a mobile app.
- If Quick Connect is not enabled, or if you select the "Manual Login" button, you will be able to select a user from the list, or manually login using your username and password.
```

**Review by mcarlton00 — APPROVED** [review]

**Pull request merged**